homeassistant/components/fyta/coordinator.py

```diff
@@ -60,7 +61,7 @@ async def _async_update_data(
 
     async def renew_authentication(self) -> bool:
         """Renew access token for FYTA API."""
-        credentials: dict[str, Any] = {}
+        credentials: Credentials
```

Comment:
Don't need this one anymore

Reply:
Really? I think without it credentials is only valid within try

Reply:
All the excepts is raising